edx_event_bus_kafka/internal/producer.py

```diff
@@ -245,22 +279,19 @@ def send(
             event_key_field: Path to the event data field to use as the event key (period-delimited
               string naming the dictionary keys to descend)
             event_data: The event data (kwargs) sent to the signal
+            event_metadata: An EventsMetadata object with all the metadata necessary for the CloudEvent spec
```

Comment:
Should we mention in code that this is optional only for rollout purposes?

Reply:
[fixed]